OmsAgent/omsagent.py

```diff
@@ -67,11 +69,16 @@ def main():
             elif re.match("^([-/]*)(update)", a):
                 dummy_command("Update", "success", "Update succeeded")
     except Exception as e:
-        err_msg = ("Failed to enable the extension with error: {0}, "
+        exit_code = 1
+        err_msg = ("Failed with error: {0}, "
```

Comment:
Will do. I'm just going to make it a single line; many lines in the file do not adhere to 80 characters/line anyway.